src/parameterizations/vertical/MOM_set_viscosity.F90

```diff
@@ -2025,6 +2026,9 @@ subroutine set_visc_init(Time, G, GV, US, param_file, diag, visc, CS, restart_CS
        diag%axesCv1, Time, 'Number of layers in viscous mixed layer at v points', 'm')
   endif
 
+  call register_restart_field_as_obsolete('Kd_turb','Kd_shear', restart_CS)
```

Comment:
@Hallberg-NOAA Should we have a single module in which to place the calls for these obsolete fields, like we do for diagnostics and parameters? It  avoids having the calls scattered throughout the code.

Reply:
As long as we only have a very few instances of these registration calls for obsolete restart names, I don't think that such a common module is necessary for this capability.  Once these calls are occurring in more than about 3 separate places scattered around the code, I think it would be time to add a new centralized repository.